plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/refactoring/preview/ChangePreview.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Shouldn't there be a remove from some dto package?

Reply:
it isn't a DTO. This class just describes information to be compared.